src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -16,16 +18,18 @@ import { PartEntityService } from '../../../core/services/models/part-entity.ser
 export class RundownComponent implements OnInit, OnDestroy {
   @Input()
   public rundown: Rundown
-
+  public isAutoNextStarted: boolean = false
   public currentEpochTime: number = Date.now()
   public remainingDurationInMsForOnAirPart?: number
   public startOffsetsInMsFromPlayheadForSegments: Record<string, number> = {}
   private rundownTimingContextSubscription?: Subscription
   private readonly logger: Logger
+  private readonly rindownEventSubscriptions: EventSubscription[] = []
```

Comment:
Small typo: Should be `rundown`, not `rindown` :)

Reply:
Done